grakn/service/Session/TransactionService.py

```diff
@@ -173,4 +173,9 @@ def __next__(self):
             self._request_next_batch(iter_res.iteratorId)
             return next(self)
         else:
-            return iter_res
+            return self._resp_converter(iter_res)
+
+    def get(self):
```

Comment:
comment saying: blocks until the next response?

==> inclear if this can be used to force blocking with `next()` so if i wanted to force blocking i could do this:
``` 
iter.get()
for ans in iter:
  iter.get()
```
or somethign?

Reply:
This should do what you are expecting, but:
```
for ans in iter:
  pass
```
Logically does the same thing. The blocking may be useful if you want to prevent some complex computation taking place until a result has been received, or to ensure that there are otherwise no errors before executing something else.